internal/overlord/servstate/handlers.go

```diff
@@ -54,9 +54,18 @@ func TaskServiceRequest(task *state.Task) (*ServiceRequest, error) {
 }
 
 var (
-	okayWait = 1 * time.Second
-	killWait = 5 * time.Second
-	failWait = 10 * time.Second
+	// okayDelay is the time to wait after starting a service before we conclude
+	// that it's running successfully.
+	okayDelay = 1 * time.Second
+
+	// killDelayDefault is the duration afforded to services for processing
+	// SIGTERM signals and shutting down cleanly if the service hasn't specified
+	// their own duration.
+	killDelayDefault = 25 * time.Second
```

Comment:
Per earlier note, seems high for the default.

Reply:
As per above.